pkg/agent/util/net_windows.go

```diff
@@ -295,3 +307,27 @@ func GetLocalBroadcastIP(ipNet *net.IPNet) net.IP {
 	binary.BigEndian.PutUint32(lastAddr, binary.BigEndian.Uint32(ipNet.IP.To4())|^binary.BigEndian.Uint32(net.IP(ipNet.Mask).To4()))
 	return lastAddr
 }
+
+func RemoveIPv4Addrs(ifaceName string) error {
```

Comment:
RemoveIPv4AddrsFromAdapter()?
And change ifName to adapterName?

Reply:
Done.